### PR TITLE
Home screen top part

### PR DESCRIPTION
**Pull request by 05Waleed**
Ticket no: [KA-08](https://shopgoswift.atlassian.net/browse/SG-8)

Description:
1. add camera button
2. add side menu button
3. add button for trending, men, women and children
4. do complete auto layout

how to test: 
Navigate from CartAnimationViewController to HomeViewController

Screenshot 

Screenshot 2023-07-27 at 18 09 30

### CODE REVIEW

File: ShopGo/HomeScreenTopPart/HomeViewController.swift [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    }
+    
+    @IBAction func womenButtonTapped(_ sender: Any) {
+    }
+    
+    @IBAction func childrenButtonTapped(_ sender: Any) {
+    }
+    
+    @IBAction func sideMenu(_ sender: Any) {
+    }
```
Comment (haroonmaqsoodch) [inline ShopGo/HomeScreenTopPart/HomeViewController.swift]: rename it to `burgerMenuButton`

File: ShopGo/HomeScreenTopPart/HomeViewController.swift [hunk 2]
```diff
@@ -0,0 +1,34 @@
+//
+//  HomeScreenTopPartViewController.swift
```
Comment (haroonmaqsoodch) [inline ShopGo/HomeScreenTopPart/HomeViewController.swift]: Rename the file to `HomeViewController`

File: ShopGo/HomeScreenTopPart/HomeViewController.swift (hunk 1 above)
Comment (05Waleed) [reply ShopGo/HomeScreenTopPart/HomeViewController.swift]: done

File: ShopGo/HomeScreenTopPart/HomeViewController.swift (hunk 2 above)
Comment (05Waleed) [reply ShopGo/HomeScreenTopPart/HomeViewController.swift]: done

**Review by haroonmaqsoodch — COMMENTED** [review]
Looks good to me, resolve the conflicts then I'll approve

**Pull request merged**